scripts/src/errors.ts

```diff
@@ -116,3 +166,155 @@ export class ChannelBusyError extends Error implements KinSdkError {
 		super('Cannot acquire a free channel.');
 	}
 }
+
+export class BadRequestError extends HorizonError {
+	readonly type: ErrorType = 'BadRequestError';
+
+	constructor(readonly errorBody: ErrorResponse, readonly title?: string) {
+		super(`Bad Request error`, errorBody, title);
+	}
+}
+
+export class InternalError extends HorizonError {
+	readonly type: ErrorType = 'InternalError';
+
+	constructor(readonly errorBody: any, readonly title?: string) {
+		super(`internal error`, errorBody, title ? title : "{'internal_error': 'unknown horizon error'}");
+	}
+}
+
+export class AccountExistsError extends HorizonError {
+	readonly type: ErrorType = 'AccountExists';
+
+	constructor(readonly errorBody: any, readonly title?: string) {
+		super(`account already exists`, errorBody, title);
+	}
+}
+
+export class LowBalanceError extends HorizonError {
+	readonly type: ErrorType = 'LowBalanceError';
+
+	constructor(readonly errorBody: ErrorResponse, readonly title?: string) {
+		super(`low balance`, errorBody, title);
+	}
+}
+
+export class AccountNotActivatedError extends HorizonError {
+	readonly type: ErrorType = 'AccountNotActivatedError';
+
+	constructor(readonly errorBody: ErrorResponse, readonly title?: string) {
+		super(`account not activated`, errorBody, title);
+	}
+}
+
+export class ResourceNotFoundError extends HorizonError {
+	readonly type: ErrorType = 'ResourceNotFoundError';
+
+	constructor(readonly errorBody: ErrorResponse, readonly title?: string) {
+		super(`resources not found`, errorBody, title);
+	}
+}
+
+export class TranslateError {
+	private readonly _resultTransactionCode?: string;
+	private readonly _resultOperationsCode?: string[];
+
+	constructor(readonly errorBody?: any) {
```

Comment:
it's kind of weird that we have a ctor that generates an error (i.e, the result), this actually can be static method, as it's pure method without state.
also, I would expect the utility to return an error object and that the user code can observer and decide if to throws (like in send transaction, that we should act differently for a specific error and throw for the rest of the cases.